login.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en" class="full-height">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <meta name="description" content="training" />
+    <meta name="author" content="Wolox" />
+    <link rel="stylesheet" type="text/css" href="application.css" />
+    <title>HTML & CSS Training</title>
+  </head>
+  <body class="full-height">
+      <main class="login-main full-height">
+        <form class="login-container">
+            <img src="src/img/LogoWolox.png" alt="" class="login-wolox-image">
+            <p class="login-label">Email</p>
+            <input type="text" class="login-input">
+            <p class="login-label">Password</p>
```

Comment:
use "label" instead of p in forms.
https://www.w3schools.com/html/html_forms.asp